src/worker/WorkerSet.ts

```diff
@@ -42,10 +48,10 @@ export default class WorkerSet extends WorkerAbstract<WorkerData> {
    * @public
    */
   public async addElement(elementData: WorkerData): Promise<void> {
+    // debug
     if (!this.workerSet) {
       throw new Error("Cannot add a WorkerSet element: workers' set does not exist");
-    }
-    if (
+    } else if (
```

Comment:
The else is not intended, the condition is expected to be checked whatever happens in the previous if(s).